### PR TITLE
Added floor_div paddle frontend

### PR DESCRIPTION
**Pull request by pranjalkar99**

### CODE REVIEW

**Comment by pranjalkar99** [comment]
Close #17665

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -1448,3 +1448,32 @@ def test_paddle_greater_than(
         frontend=frontend,
         on_device=on_device,
     )
+
+
+# floor_divide
+def test_paddle_floor_divide(
+    dtypes_and_x,
```
Comment (YushaArif99) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1455]: I think you've forgotten to use the `@frontend_method` decorator here. You can look at the other examples above to see how to correctly write the test function 🙂

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @pranjalkar99 

Apologies for the late review. Have been extremely busy with work recently and so haven't had the time to review PRs 😓 

Could you kindly address the abovementioned issue and re-request a review once all tests are passing. Thanks! 🙂

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**